web-apps/genesys-spark-examples/src/utils/generateIcons.js

```diff
@@ -223,7 +223,6 @@ const icons = [
       'device-cellphone',
       'device-headphones',
       'device-laptop',
-      'device-pc',
```

Comment:
I would comment this out with an explanation so that in a few months someone does not think it is missing by mistake and add it back in.

Reply:
Done